components/google_drive/google_drive.app.js

```diff
@@ -26,11 +31,51 @@ module.exports = {
         return this._listDriveOptions(nextPageToken);
       },
     },
+    folderId: {
+      type: "string",
+      label: "Folder",
+      description: "The folder in the drive",
+      options({
+        prevContext,
+        drive,
+        baseOpts = {
+          q: `mimeType = '${GOOGLE_DRIVE_FOLDER_MIME_TYPE}'`,
+        },
+      }) {
+        const { nextPageToken } = prevContext;
+        return this.listDriveFilesOptions(drive, nextPageToken, baseOpts);
```

Comment:
There is some way to show the entire folder path? I think we may have a scenario where the user has two folders with the same name but in different paths. Does it make sense?